src/MCServersManager.js

```diff
@@ -12,6 +12,7 @@ const EULA_TEMPLATE_FILENAME = 'template.eula.txt';
 const EULA_FILENAME = 'eula.txt';
 const SERVER_PROPERTIES_FILENAME = 'server.properties';
 
+console.log(MCVersionsManager);
 const mcvm = new MCVersionsManager();
```

Comment:
I think it makes sense here to use your fancy MCVMInterface

Reply:
Will add when I update MCSM to TS